pi4j-core/src/main/java/com/pi4j/io/pwm/impl/DefaultPwmConfig.java

```diff
@@ -93,7 +93,7 @@ protected DefaultPwmConfig(Map<String,String> properties){
 
         // load optional pwm duty-cycle from properties
         if(properties.containsKey(DUTY_CYCLE_KEY)){
-            this.dutyCycle = Float.parseFloat(properties.get(DUTY_CYCLE_KEY));
+            this.dutyCycle = Integer.parseInt(properties.get(DUTY_CYCLE_KEY));
```

Comment:
Would be better to have a generic method, e.g. properties.getInt(string, int default), that has exception handling if the value can't be converted to integer and return the default value in such case